tests/core/test_leader_elector.py

```diff
@@ -0,0 +1,166 @@
+# stdlib
+import datetime
+import time
+
+# 3rd party
+import mock
+
+# project
+from utils.kubernetes import LeaderElector
+from tests.core.test_kubeutil import KubeTestCase
+
+
+HEALTH_ENDPOINT = '/healthz'
```

Comment:
Shoud you import the contants from the `LeaderElector` class instead of copying them?